test/event_handler_test.py

```diff
@@ -82,6 +88,7 @@ def test_reads_messages_from_queue_with_key_from_s3(self):
 
     def test_reads_fraud_events_from_queue(self):
         self.__setup_s3()
+        responses.add_passthru(f'https://{SPLUNK_HEC_HOST}')
```

Comment:
Would have preferred a mock for Splunk backend since this is a unit test and it's a side-effect, but won't block the PR because of this.